src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/Recipe.cs

```diff
@@ -480,5 +486,61 @@ private void ConfigureBeanstalkEnvironment(Configuration settings, string beanst
                 VersionLabel = ApplicationVersion.Ref,
             }));
         }
+
+        /// <summary>
+        /// When deploy a self contained deployment bundle Beanstalk needs a Procfile to tell the environment what process to start up.
```

Comment:
replace "deploy" by "deploying" and add a comma after "bundle"

Reply:
Done